app/views/shared/views/_prov_dialog.html.haml

```diff
@@ -297,7 +335,7 @@
             :label                      => _("Custom Specification"),
             :prefix                     => "/miq_request/",
             :keys                       => keys})
-      - if (!@options && @edit && @edit[:new] && @edit[:new][:sysprep_enabled] && @edit[:new][:sysprep_enabled][0] && @edit[:new][:sysprep_enabled][0] != "disabled") || (!@edit && @options && @options[:sysprep_enabled] && @options[:sysprep_enabled][0] && @options[:sysprep_enabled][0] != "disabled")
+      - if ((!@options && @edit && @edit[:new] && @edit[:new][:sysprep_enabled] && @edit[:new][:sysprep_enabled][0] && @edit[:new][:sysprep_enabled][0] != "disabled") || (!@edit && @options && @options[:sysprep_enabled] && @options[:sysprep_enabled][0] && @options[:sysprep_enabled][0] != "disabled")) && !wf.kind_of?(ManageIQ::Providers::Redhat::InfraManager::ProvisionWorkflow)
```

Comment:
this line is getting even longer than before

can you extract that to a (helper) method?

Reply:
@martinpovolny Where do you propose to extract this condition to? This condition needs to be checked for ovirt and vmware provider.